ops/convex-real/manage.ts

```
@@ -178,26 +204,101 @@ function writeTestEnvFile(): string {
+  const backupPath = `${envFilePath}.convex-real-backup-${process.pid}-${Date.now()}`;
+  renameSync(envFilePath, backupPath);
+  try {
+    return await operation();
+  } finally {
+    if (existsSync(envFilePath)) {
+      const unexpectedPath = `${backupPath}.unexpected`;
+      renameSync(envFilePath, unexpectedPath);
+      renameSync(backupPath, envFilePath);
+      throw new Error(
+        `Self-hosted Convex command recreated ${envFilePath}; original restored and unexpected file moved to ${unexpectedPath}`
+      );
```

Biome (2.5.1)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)